### ISSUE TITLE
Expose information about Git

### ISSUE THREAD

**Issue by jennybc** [rstudio/rstudioapi#89]
It would be very handy to know the IDE's current impression of:

  * `git_available()`
  * `git_version()`
  * `git_path()`

similar to what rmarkdown does for Pandoc, `rmarkdown::pandoc_available()` and `rmarkdown::pandoc_version()`.

**Comment by jdblischak** [comment]
I agree. It would be very useful to know if the user is able to use Git via RStudio. For example, I found 20 `vcs*` commands that can be executed with `rstudioapi::executeCommand()` ([source](https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml)), but how can I know if these will work? As far as I can tell, `executeCommand()` always returns `NULL` regardless of whether the command succeeded or not.

xref: https://github.com/rstudio/rstudioapi/pull/130#issuecomment-470245700
